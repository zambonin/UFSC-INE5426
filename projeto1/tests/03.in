int i = 0, j
float f = 1.1
bool b = true
j = [int] [int] i + f
i = [int] j
b = b & [bool] f
f = ([float] b) + 0.0

int l=0
float k=0

k = 12 + l - k
l = [int] ([int] 12.3) / k

