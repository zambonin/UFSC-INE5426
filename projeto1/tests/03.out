int var: i = 0, j
float var: f = 1.1
bool var: b = true
= j [int] [int] + [float] i f
= i [int] j
= b & b [bool] f
= f + [float] b 0.0
int var: i = 0
float var: f = [float] 0
= f - [float] + 12 i f
= i [int] / [float] [int] 12.3 f
